### DISCUSSION TITLE
Community meeting Jan 5, 2021

### DISCUSSION THREAD

**Discussion by carlisia** [vmware-tanzu/velero#3225] [Community meetings]
### Status Updates
-    carlisia
    -    Back from 2 weeks PTO 
    -    This week will be PR reviews and addressing code reviews on my PRs
- bridget
    - Catching up after PTO
    - On community support this week
    - Will pick up work on multiple credentials support again
        - [PR #2403](https://github.com/vmware-tanzu/velero/pull/2403) needs another review
- nrb
    - Also catching up after PTO - need to catch up on opened issues and outstanding PRs
### Discussion Topics
- carlisia
    - are we ready to start with the GitHub Discussion? I started a list of potential categories here: https://hackmd.io/OFFTp_RIRmOQ7xwhxkv0yA

### Contributor Shoutouts

**Comment by jonasrosland** [comment]
YouTube link of the recording: https://youtu.be/sQL_5lvR2nw
